api/HTMLImageElement.json

```
@@ -669,6 +669,56 @@
           }
         }
       },
+      "loading": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "77"
+            },
+            "chrome_android": {
+              "version_added": "77"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": "75"
+            },
+            "firefox_android": {
+              "version_added": false,
+              "notes": "See <a href='https://bugzil.la/1542784'>bug 1542784</a>"
```

This links to a fixed bug. I think we should not add it. I don't see anything useful in there for Fx Android.

```suggestion
              "version_added": false
```